gateway/ln-gateway/src/lnd.rs

```diff
@@ -749,3 +756,30 @@ fn route_hints_to_lnd(
         })
         .collect()
 }
+
+fn wire_features_to_lnd_feature_vec(features_wire_encoded: &[u8]) -> anyhow::Result<Vec<i32>> {
```

Comment:
I don't think it's a blocker for merging, but should we make a followup that adds some test vectors to ensure we're correctly translating?

Reply:
Good point, added a test.